src/common/offline_transformations/include/mask_attribute.hpp

```diff
@@ -180,6 +180,26 @@ class Mask : public std::vector<std::set<uint64_t>>, public std::enable_shared_f
         return result_mask;
     }
 
+    void copy_and_slice_mask_from(const ngraph::Mask* const mask,
+                                  int64_t axis,
+                                  uint64_t split_start,
+                                  uint64_t split_end) {
+        if (size() < mask->size())
+            resize(mask->size());
+        for (size_t i = 0; i < size(); i++) {
+            if (i == axis) {
+                std::set<uint64_t> dst_set;
+                const auto& src_set = (*mask)[i];
```

Comment:
please use `at` instead of `*` here and elsewhere

Reply:
I got your concern and I understand, that `*[]` is faster and clear, but I suggest to keep `at` because
1) I would like to keep code style consistent and ether use `at` or `*[]` everywhere.
2) Pruning Transformation works quite fast, and `at` calls are not a bottleneck in any case.
If you would like to use `*[]`, please open a PR where all `at` calls (In the pruning transformation we can replace all of them, I believe)  are replaced by `*[]`

@jane-intel, what do you think about your this comment?

Reply:
I agree with both solutions (at or *[]) but it's better to unify this code and use only one implementation.
Could you use `at` here?
Or could you create a new jira ticket to refactor this transformation to use `*[]`?